diff --git a/node_modules/react-quill/lib/index.js b/node_modules/react-quill/lib/index.js
index 23c3b84..4a51e6d 100644
--- a/node_modules/react-quill/lib/index.js
+++ b/node_modules/react-quill/lib/index.js
@@ -528,7 +528,7 @@
 	},
 
 	getEditingArea: function(){
-		return ReactDOM.findDOMNode(this.editingArea);
+		return this.editingArea;
 	},
 
 	getEditor: function(){
@@ -700,7 +700,7 @@
 			this.state.value
 		);
 
-		var editingArea = ReactDOM.findDOMNode(this.editingArea);
+		var editingArea = this.editingArea;
 		var editor = this.instantiateEditor(
 			editingArea,
 			this.getEditorConfig()
@@ -724,7 +724,7 @@
 	},
 
 	setEditingAreaRef: function(ref) {
-		this.editingArea = ref;
+		this.editingArea = ref;	
 	},
 
 	render: function() {
diff --git a/node_modules/react-quill/lib/mixin.js b/node_modules/react-quill/lib/mixin.js
index f471e42..0c2e783 100644
--- a/node_modules/react-quill/lib/mixin.js
+++ b/node_modules/react-quill/lib/mixin.js
@@ -8,7 +8,7 @@
 	},
 
 	getEditingArea: function(){
-		return ReactDOM.findDOMNode(this.editingArea);
+		return this.editingArea;
 	},
 
 	getEditor: function(){
@@ -129,7 +129,7 @@
 			this.state.value
 		);
 
-		var editingArea = ReactDOM.findDOMNode(this.editingArea);
+		var editingArea = this.editingArea;
 		var editor = this.instantiateEditor(
 			editingArea,
 			this.getEditorConfig()
@@ -153,7 +153,7 @@
 	},
 
 	setEditingAreaRef: function(ref) {
-		this.editingArea = ref;
+		this.editingArea = ref;	
 	},
 
 	/*
