diff --git a/node_modules/react-quill/lib/index.js b/node_modules/react-quill/lib/index.js
index 9288824..2da2686 100644
--- a/node_modules/react-quill/lib/index.js
+++ b/node_modules/react-quill/lib/index.js
@@ -481,7 +481,7 @@
 	},
 
 	getEditingArea: function() {
-		return ReactDOM.findDOMNode(this.editingArea);
+		return this.editingArea;
 	},
 
 	getEditor: function() {
@@ -628,7 +628,9 @@
 			this.state.value,
 			this.getEditor()
 		);
-		this.editingArea = editor.root;
+		this.editingArea = editor.root;	
+		this.editingArea.ref = (ref) => { this.editingArea = ref };
+
 	},
 
 	destroyEditor: function() {
